records_mover/db/postgres/copy_options/csv.py

```diff
@@ -70,12 +71,21 @@ def postgres_copy_options_csv(unhandled_hints: Set[str],
     #  specified, non-NULL values will be quoted in all columns. This
     #  option is allowed only in COPY TO, and only when using CSV
     #  format.
-    #
-
-    if hints['quoting'] != 'minimal':
-        cant_handle_hint(fail_if_cant_handle_hint, 'quoting', hints)
+    if mode is CopyOptionsMode.LOADING:
+        if hints['quoting'] != 'minimal':
+            cant_handle_hint(fail_if_cant_handle_hint, 'quoting', hints)
+        else:
+            quiet_remove(unhandled_hints, 'quoting')
+    elif mode is CopyOptionsMode.UNLOADING:
+        # TODO: Are we sure this doesn't support minimal by default instead of None by default?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker